src/vs/workbench/contrib/terminal/browser/media/shellIntegration-bash.sh

```diff
@@ -13,7 +13,9 @@ VSCODE_SHELL_INTEGRATION=1
 # Run relevant rc/profile only if shell integration has been injected, not when run manually
 if [ "$VSCODE_INJECTION" == "1" ]; then
 	if [ -z "$VSCODE_SHELL_LOGIN" ]; then
-		. ~/.bashrc
+		if [ -f ~/.bashrc ]; then
```

Comment:
better to use `[ -r ~/.bashrc ]`

Reply:
Thanks https://github.com/microsoft/vscode/pull/171495